draft-shafranovich-rfc4180-bis.md

```diff
@@ -150,31 +167,47 @@ of {{?RFC7231}}). However, some implementations MAY use other values.
 The ABNF grammar (as per {{!RFC5234}}) appears as follows:
 
 ~~~~~~~~~~
-file = *(record linebreak)
+file = *((comment / record) linebreak)
+
+comment = hash *comment-data
 
-record = field *(COMMA field)
+record = first-field *(comma field)
 
 linebreak = CR / LF / CRLF
 
+first-field = (escaped / first-non-escaped)
+
 field = (escaped / non-escaped)
 
-escaped = DQUOTE *(TEXTDATA / COMMA / CR / LF / 2DQUOTE) DQUOTE
+escaped = DQUOTE *(textdata-with-hash / comma / CR / LF / 2DQUOTE) DQUOTE
```

Comment:
Should we change "CR / LF" to read "linebreak" instead, and this will include everything

Reply:
It wouldn't hurt but it is superfluous as every character in this sequence group can repeat anyway. The only exception is 2DQUOTE because it has to. Within the definition of `file` this is different as we have to describe the linebreak as a list of potential characters or character sequence without the ability to repeat. So, changing this here wouldn't be wrong but wouldn't help either and might cause headache for implementors. What do you think?